vcd/datasource_vcd_network_test.go

```diff
@@ -285,6 +296,10 @@ output "network_name" {
   value = data.vcd_network_direct.{{.NetworkName}}.name
 }
 
+output "network_description" {
```

Comment:
I did that for a specific reason: using output fields will allow me to use the generated HCL script to see the result when running it with `terraform` tool. If I don't put them, I will have to edit the file to see the result.  Since I had to do this operation several times when I was developing, I thought that I get the best of two worlds with this approach: result in the integrated tests, and visible output when running manually.